spec/requests/api/v1/sessions/destroy_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'DELETE api/v1/users/sign_out', type: :request do
+  let(:user) { create(:user) }
+
+  it 'returns a succesful response' do
+    delete destroy_user_session_path, headers: auth_headers, as: :json
+    expect(response).to have_http_status(:success)
+  end
+
+  it 'can not log out twice' do
```

Comment:
I'd prefer adding a context `'when the user is not logged in'` then you can test that the response is not found. I don't like to call twice the endpoint.